tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -145,25 +139,57 @@
                     </ic:EventTriggerBehavior>
                 </i:Interaction.Behaviors>
             </AutoSuggestBox>
-            <!-- Sort field -->
-            <TextBlock x:Uid="SortByTextBlock" VerticalAlignment="Center" Margin="50 0 10 0"/>
-            <ComboBox x:Uid="SortSelectionComboBox" x:Name="SortSelectionComboBox" Margin="0 3 0 0" Height="35">
-                <ComboBoxItem x:Uid="SortByName"/>
-                <ComboBoxItem x:Uid="SortByAltName"/>
-                <i:Interaction.Behaviors>
-                    <ic:EventTriggerBehavior EventName="SelectionChanged">
-                        <ic:InvokeCommandAction
-                            Command="{x:Bind ViewModel.SortHandlerCommand}"
-                            CommandParameter="{x:Bind ((ComboBoxItem)SortSelectionComboBox.SelectedItem).Content, Mode=OneWay}" />
-                    </ic:EventTriggerBehavior>
-                </i:Interaction.Behaviors>
-            </ComboBox>
+            <StackPanel Grid.Column="2" Orientation="Horizontal" HorizontalAlignment="Right">
+                <!-- Provider field -->
+                <TextBlock x:Uid="ProviderTextBlock" VerticalAlignment="Center" Margin="50 0 5 0"/>
+                <ComboBox x:Uid="ProviderSelectionComboBox" x:Name="ProviderSelectionComboBox" Margin="0 3 5 0"
+                          ItemsSource="{x:Bind ViewModel.Providers}" SelectedIndex="{x:Bind ViewModel.SelectedProviderIndex, Mode=TwoWay}">
+                    <i:Interaction.Behaviors>
+                        <ic:EventTriggerBehavior EventName="SelectionChanged">
+                            <ic:InvokeCommandAction Command="{x:Bind ViewModel.ProviderHandlerCommand}" />
+                        </ic:EventTriggerBehavior>
+                    </i:Interaction.Behaviors>
+                </ComboBox>
+                <!-- Sort field -->
+                <TextBlock x:Uid="SortByTextBlock" VerticalAlignment="Center" Margin="10 0 5 0"/>
+                <ComboBox x:Uid="SortSelectionComboBox" x:Name="SortSelectionComboBox" Margin="0 3 13 0"
+                          SelectedIndex="{x:Bind ViewModel.SelectedSortIndex, Mode=TwoWay}">
+                    <ComboBoxItem x:Uid="SortByName"/>
+                    <ComboBoxItem x:Uid="SortByNameDesc"/>
+                    <ComboBoxItem x:Uid="SortByAltName"/>
+                    <ComboBoxItem x:Uid="SortByAltNameDesc"/>
+                    <ComboBoxItem x:Uid="SortByLastConnected"/>
+                    <i:Interaction.Behaviors>
+                        <ic:EventTriggerBehavior EventName="SelectionChanged">
+                            <ic:InvokeCommandAction Command="{x:Bind ViewModel.SortHandlerCommand}"/>
+                        </ic:EventTriggerBehavior>
+                    </i:Interaction.Behaviors>
+                </ComboBox>
+                <!--  'Sync' button -->
+    <!-- ToDo: Revisit after confirming glyphs to be used -->

```

Comment:
This still needs a Todo?

Reply:
I mistook this to be someone else's todo. Will remove in a later PR.

Reply:
Removed.